### PR TITLE
49 Add more SL UI variations

### PR DESCRIPTION
**Pull request by zamorastp**

### CODE REVIEW

**Comment by zamorastp** [comment]
See also https://github.schibsted.io/spt-identity/identity-sdk-ios-tracking/pull/37

**Comment by zamorastp** [comment]
Simulator Screen Shot - iPhone 13 - 2022-03-24 at 13 52 54
Simulator Screen Shot - iPhone 13 - 2022-03-24 at 13 53 04
Simulator Screen Shot - iPhone 13 - 2022-03-24 at 13 53 21

File: Sources/AccountSDKIOSWeb/Resources/da.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,8 +1,11 @@
-"SimplifiedWidget.continueToLogIn" = "Fortsæt med at logge ind";
+"SimplifiedWidget.logInWithOneClick" = "Log ind med et enkelt klik";
+"SimplifiedWidget.loginIncentive" = "Du er allerede logget ind på en af
+Schibsteds ydelser.";
```
Comment (clausjoergensen) [inline Sources/AccountSDKIOSWeb/Resources/da.lproj/Localizable.strings:3]:
```suggestion
"SimplifiedWidget.loginIncentive" = "Du er allerede logget ind på en af Schibsteds ydelser.";
```

Comment (clausjoergensen) [reply Sources/AccountSDKIOSWeb/Resources/da.lproj/Localizable.strings]: Oh yeah, I was mostly concerned about the extra newline, which seemed like a typo. It's better to use `\n` in strings files than letting it span multiple lines.

Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Resources/da.lproj/Localizable.strings]: Good point!

**Review by clausjoergensen — APPROVED** [review]

**Review by mbanasiewicz — APPROVED** [review]

**Review by oscarsilver — APPROVED** [review]
LGTM

**Pull request merged**